pom.xml

```diff
@@ -76,17 +76,47 @@
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-protobuf</artifactId>
-			<version>1.20.0</version>
+			<version>1.40.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-stub</artifactId>
-			<version>1.20.0</version>
+			<version>1.40.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-netty</artifactId>
-			<version>1.20.0</version>
+			<version>1.40.0</version>
+		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-codec</artifactId>
+			<version>4.1.61.Final</version>
+		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-codec-http2</artifactId>
+			<version>4.1.61.Final</version>
+		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-transport</artifactId>
+			<version>4.1.61.Final</version>
+		</dependency>
```

Comment:
this is adding explicit dependencies for netty pacakges - is this expected? Can you please check which major versions we were using earlier?

Reply:
We used 4.1.30 before. After updating grpc to 1.40.0, the netty packages used older versions which did not resolve the vulnerability in Component Governance. 
So some netty packages had to be explicitly updated here.